pkgs/by-name/gk/gk-cli/package.nix

```diff
@@ -0,0 +1,122 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+  writeShellScript,
+  curl,
+  jq,
+  common-updater-scripts,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gk-cli";
+  version = "2.1.0";
+
+  src = (
+    finalAttrs.passthru.sources.${stdenv.system}
+      or (throw "gk-cli ${finalAttrs.version} unsupported system: ${stdenv.system}")
+  );
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm555 gk -t $out/bin/
+
+    installShellCompletion --bash ./**/gk.bash
+    installShellCompletion --fish ./**/gk.fish
+    installShellCompletion --zsh ./**/_gk
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/gk \
+      --prefix PATH : "${lib.makeBinPath [ coreutils ]}"
+  '';
+
+  passthru = {
+    sources =
+      let
+        base_url = "https://github.com/gitkraken/gk-cli/releases/download/v${finalAttrs.version}/gk_${finalAttrs.version}_";
+      in
+      {
+        aarch32-linux = fetchzip {
+          # i don't know which kind of ARM it is
```

Comment:
i will remove this when i will squash before the merge

but i would like to know how to know which kind of ARM it is